### ISSUE TITLE
[BUG] RSS shortcake has H2 rendering inside list

### ISSUE THREAD

**Issue by uwtwthorpe** [uweb/uw_wp_theme#91]
## Describe the bug
The RSS shortcode has a structure that is not accessible. The code output has a header and list. The header, however, appears inside the list. That is, there is a UL tag and then immediately after an H2 tag. The H2 tag should come before the UL tag.

## Describe your environment
I'm on a MacBook Pro (14") running Sequoia 15.5 using Safari 18.5 (20621.2.5.11.8).

## Steps to reproduce
1. Go to https://www.washington.edu/about/
2. Scroll down to "UW News releases"
3. View the inspector for the section, specifically looking at the list of stories under the header

## Expected behavior
The H2 tag would appear before the UL tag.

## Current Behavior
The H2 tag is located inside the UL tag, rendering non-accessible HTML.

## Possible Solution

Move the H2 so that it renders before the list.

**Comment by tjsuw** [comment]
This has been updated in the theme as of [3.8](https://github.com/uweb/uw_wp_theme/releases/tag/3.8.0).
